www/header.php

```diff
@@ -23,12 +23,14 @@
  *
  */
 -->
+<!--removeIf(GENINDEXDEV)-->
```

Comment:
Some comments are added which are used for preprocessing if generating a frontend only development index.html.
Doesn't break anything.